### PR TITLE
Olha's solutions and comments

### PR DESCRIPTION
**Pull request by olhanotolga**

### CODE REVIEW

File: Olha/index.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+let percentageComplete = 40;
+
+switch (true) {
+	case (percentageComplete < 30):
+		console.log("Still a long way to go");
+		break;
+	case (percentageComplete <= 50 && percentageComplete >= 30):
+		console.log("Slowly getting there");
+		break;
+	case (percentageComplete <= 51 && percentageComplete >= 80):
```
Comment (ciamac-da) [inline Olha/index.js]: Line 85 ==> In this case the range is equal 51 and  bigger than  51 AND also is equal 80 and bigger than 80
but it should be between 51 and 80

File: Olha/index.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+		console.log("You can do it!");
+		break;
+	case (percentageComplete <= 81 && percentageComplete >= 99):
```
Comment (ciamac-da) [inline Olha/index.js]: Line 88 ==> In this case the range is equal 81 and smaller than 51 AND also is equal 99 and bigger than 99
but it should be between 81 and 99

**Review by ciamac-da — CHANGES_REQUESTED** [review]

File: Olha/index.js (hunk 2 above)
Comment (olhanotolga) [reply Olha/index.js]: Thanks for noticing! Fixed!

**Review by ciamac-da — APPROVED** [review]
💯

**Pull request merged**